### ISSUE TITLE
Error when provisioning a cluster using rancher

### ISSUE THREAD

**Issue by pdaccache** [nutanix/docker-machine#52]
I am receiving the below error once i try to provision using rancher, the VMs gets created and IPs assigned but it stuck on the below error:

Cluster must have at least one etcd plane host: failed to connect to the following etcd host(s) [10.50.29.184]

**Comment by tuxtof** [comment]
normally the Nutanix Driver job is done at this step if vm are created and they get an IP

i think best here is to open a ticket against rancher side

or maybe you have more information who allow you to suspect the issue is on nutanix driver side ??

happy to help in this case

**Comment by pdaccache** [comment]
Yes the VMs gets created and assigned an IP, when i open rancher logs i see the above error. When i download node keys from rancher UI in order to connect to it using ssh to check docker logs i am not able to connect with error wrong keys.
Could it be related to the ubuntu image being used?? I am using the ubuntu 22.04 amd64.img

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by tuxtof** [comment]
can you share the exact url where you get your `ubuntu 22.04 amd64.img`

when you download the rancher key did you try the connection with the root user ?

you can also try to add you own key in the node template / cloud init with a different username

**Comment by tuxtof** [comment]
Yes this is the image i use too

please test with a custom cloudinit

**Comment by pdaccache** [comment]
I have tried and still i am not able to login to the nodes (i have tried
using SSH private key and using a password for the ubuntu user).
What else i should try?

**Comment by tuxtof** [comment]
can you paste your cloud-init here , did you cloud-init works with a singleubuntu vm ?
also a screenshot of your nutanix node template ?

**Comment by pdaccache** [comment]
i was able to SSH into the control plane node, and the only error received
is as below:

loopback address found in /etc/resolv.conf, please refer to the
documentation

i have added DNS configuration into cloud-init and still same issue and
same error in docker logs.

**Comment by tuxtof** [comment]
if you are able to connect to the node in ssh , all seems OK on the nutanix driver side

inside your control plane node are you able to launch a docker command ?
this is the first install step rancher do after vm provisonning

now you need to search on the provisionning step and the best is to ask help on the rancher side repo

**Comment by pdaccache** [comment]
yes i am able to run docker commands, i was able to fix the DNS issue but
now i am receiving a new error:
ERROR: The environment variable CATTLE_CA_CHECKSUM is set but there is no
CA certificate configured at https://domainname/v3/settings/cacerts.

**Comment by tuxtof** [comment]
@pdaccache looks like a rancher isse
